feeds/utils.R

```diff
@@ -19,10 +19,14 @@ use_pat <- function() {
 use_pat()
 
 
-get_list_repos <- function(org, ignore_archived = FALSE,
-                           ignore_pattern = NULL, ...) {
-
-  init_res  <- gh::gh("GET /orgs/:org/repos", org = org)
+get_list_repos <- function(org,
+                           org_is_user = FALSE,
+                           ignore_archived = FALSE,
+                           ignore_pattern = NULL,
+                           ...) {
+
+  init_res  <- if(org_is_user) gh::gh("GET /users/:org/repos", org = org)
+               else gh::gh("GET /orgs/:org/repos", org = org)
```

Comment:
@tkphd Unfortunately, this is failing, see https://github.com/hpc-carpentry/hpc-carpentry.github.io/actions/runs/3386633494/jobs/5626289198#step:9:47. I think these are the wrong way around?

Reply:
Fixed in #65